### ISSUE TITLE
error while grabbing the id and key

### ISSUE THREAD

**Issue by Mingels1** [AMoo-Miki/homebridge-tuya-lan#237]
I followed the instructions. Result was the following error
<img width="648" alt="Schermafbeelding 2021-04-18 om 14 51 49" src="https://user-images.githubusercontent.com/78409667/115146289-23039100-a056-11eb-994b-6a0014cf3ac7.png">

**Comment by Raiden38** [comment]
Yeah, I also have those errors. Each time the app connects, it says: Failed to intercept secure communications. This could happen due to bad CA certificate.

**Comment by SmolSoftBoi** [comment]
Also getting the same error.

**Comment by LAOCDev** [comment]
Any updates on this error? Had the same issue today. Also tried the method to get the id/key via their web portal but they are requiring a subscription to create projects

**Comment by alexpad16** [comment]
Same error also on my side.

**Comment by mikkhait** [comment]
are you guys on iOS15?

**Comment by alexpad16** [comment]
> are you guys on iOS15?

No, I am using iOS 14.6. Anyway in the meantime I wanted to use the tuya-convert and managed to brick the smart plug

**Comment by LAOCDev** [comment]
I actually tried from iPadOS on the latest (non beta) version.

**Comment by wri-matt** [comment]
Same problem here. Not on iOS 15.

**Comment by ttambo** [comment]
Same error here, used this a year ago with no issues. Using the Kogan Smarter Home app

**Comment by victorczon** [comment]
Same issue but I'm on iOS 15 Beta

Failed to intercept secure communications. This could happen due to bad CA certificate.

**Comment by edwerde** [comment]
Having the same problem on iOS 15
Any update on this?

**Comment by amdjml** [comment]
I am on iOS15 and when opening the Tuya app, I get the following error:

Network is unavailable, please check the network. (Only 2 options available, OK & Help) and in console of homebridge i see the following errors:

`Intercepted certificate request
Error: ERR_SSL_UNSUPPORTED_PROTOCOL [Error: 140427578001280:error:14209102:SSL routines:tls_early_post_process_client_hello:unsupported protocol:../deps/openssl/openssl/ssl/statem/statem_srvr.c:1686:
] {
  library: 'SSL routines',
  function: 'tls_early_post_process_client_hello',
  reason: 'unsupported protocol',
  code: 'ERR_SSL_UNSUPPORTED_PROTOCOL'
}
Error: ERR_SSL_UNSUPPORTED_PROTOCOL [Error: 140427578001280:error:14209102:SSL routines:tls_early_post_process_client_hello:unsupported protocol:../deps/openssl/openssl/ssl/statem/statem_srvr.c:1686:
] {
  library: 'SSL routines',
  function: 'tls_early_post_process_client_hello',
  reason: 'unsupported protocol',
  code: 'ERR_SSL_UNSUPPORTED_PROTOCOL'
}
Error: ERR_SSL_UNSUPPORTED_PROTOCOL [Error: 140427578001280:error:14209102:SSL routines:tls_early_post_process_client_hello:unsupported protocol:../deps/openssl/openssl/ssl/statem/statem_srvr.c:1686:
] {
  library: 'SSL routines',
  function: 'tls_early_post_process_client_hello',
  reason: 'unsupported protocol',
  code: 'ERR_SSL_UNSUPPORTED_PROTOCOL'
}`

**Comment by edwerde** [comment]
Managed to get the Id and local key using this
https://www.youtube.com/watch?v=FpY-xsY-pZ8

**Comment by 1337hero** [comment]
> Managed to get the Id and local key using this https://www.youtube.com/watch?v=FpY-xsY-pZ8

video is private

**Comment by epheterson** [comment]
Seeing the same error, any idea? I got an ID and Key for the official Tuya Homebridge plugin using the following method. Is that the same key?

https://github.com/iRayanKhan/homebridge-tuya/wiki/Get-Local-Keys-for-Your-Devices

**Comment by 1337hero** [comment]
> Seeing the same error, any idea? I got an ID and Key for the official Tuya Homebridge plugin using the following method. Is that the same key?
> 
> https://github.com/iRayanKhan/homebridge-tuya/wiki/Get-Local-Keys-for-Your-Devices

So a few weeks ago when I posted the comment above yours, I was able to get my ID and Key via the local lan method with the Certificate and whatnot. Today however, for a new device that did not work. At all. I got errors similar to the OP's post. It seems others on iOS15 are now having that issue. 

However, I was able to 100% get the ID and Key via this method. So this is solid. 

https://github.com/iRayanKhan/homebridge-tuya/wiki/Get-Local-Keys-for-Your-Devices
